src/hermit.rs

```diff
@@ -37,10 +37,16 @@ pub fn prepare_environment(project_dir: &Path, hermit_env_path: &Option<PathBuf>
 	}
 }
 
+pub enum NetworkConfig {
+	VirtIO(network::VirtioNetworkConfig),
+	UserNetwork(u16),
+	None,
+}
```

Comment:
I think this should either represent the Network backend (`tap|bridge|user|l2tpv3|vde|netmap|vhost-user|socket`) or the emulated network device (`rtl8139`, `e1000`, `virtio-net-pci`, etc.), but not mix those two dimensions.

https://www.qemu.org/docs/master/system/invocation.html#hxtool-5

Reply:
I agree and fixed it.